install/install.go

```diff
@@ -550,7 +549,7 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	return nil
 }
 
-func autoDetectEnvironment(c *k8s.Client) (name string) {
+func AutoDetectEnvironment(c *k8s.Client) (name string) {
```

Comment:
Can you add a simple comment in the format of `// AutoDetectEnvironment detect the environment for a given k8s context`